pkg/controller/buildrun/buildrun_controller.go

```diff
@@ -173,6 +173,18 @@ func (r *ReconcileBuildRun) Reconcile(request reconcile.Request) (reconcile.Resu
 		if buildRun.Status.BuildSpec == nil {
 			buildRun.Status.BuildSpec = &build.Spec
 		}
+
+		if len(lastTaskRun.Status.Steps) > 0{
+			for iter := range lastTaskRun.Status.Steps {
+				buildStep := lastTaskRun.Status.Steps[iter]
+				buildRun.StageInfo.Name = buildStep.Name
+				buildRun.StageInfo.Status = buildStep.Terminated.Reason
+				buildRun.StageInfo.StartTime = buildStep.Terminated.StartedAt
+				//buildRun.StageInfo.DurationMilliSeconds = buildStep.Terminated.FinishedAt
```

Comment:
Can you help remove this non-used `DurationMilliSeconds` setting and also add ut to cover that?

BTW, can you help provide an example what the result so that we can see what it looks like?

Thanks!